runtests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#rm -rf env
+virtualenv env
+. env/bin/activate
+pip install -e .
+pip install nose mock coverage
+nosetests --with-coverage --nocapture calico.openstack.test.test_plugin:TestPlugin
```

Comment:
I tried a Makefile, but it doesn't work, because all the above commands from '. env/bin/activate' onwards have to be in the same shell.

So, I've kept this as a script but tidied it up and removed the nocapture and plugin-specific testing - let me know what you think when you see the new version.